packages/@aws-cdk/aws-lambda-destinations/lib/lambda.ts

```diff
@@ -1,22 +1,65 @@
+import * as events from '@aws-cdk/aws-events';
+import * as targets from '@aws-cdk/aws-events-targets';
 import * as lambda from '@aws-cdk/aws-lambda';
 import { Construct } from '@aws-cdk/core';
+import { EventBridgeDestination } from './event-bridge';
+
+/**
+ * Options for a Lambda destination
+ */
+export interface LambdaDestinationOptions {
+  /**
+   * Whether the destination function receives only the `responsePayload` of
+   * the origin function.
+   *
+   * @default false The destination function receives the full invocation record.
+   */
+  readonly responseOnly?: boolean;
+}
 
 /**
  * Use a Lambda function as a Lambda destination
  */
 export class LambdaDestination implements lambda.IDestination {
-  constructor(private readonly fn: lambda.IFunction) {
+  constructor(private readonly fn: lambda.IFunction, private readonly options: LambdaDestinationOptions = {}) {
   }
 
   /**
    * Returns a destination configuration
    */
-  public bind(_scope: Construct, fn: lambda.IFunction): lambda.DestinationConfig {
-    // deduplicated automatically
-    this.fn.grantInvoke(fn);
+  public bind(scope: Construct, fn: lambda.IFunction, options?: lambda.DestinationOptions): lambda.DestinationConfig {
+    // Normal Lambda destination (full invocation record)
+    if (!this.options.responseOnly) {
+      // deduplicated automatically
+      this.fn.grantInvoke(fn);
+
+      return {
+        destination: this.fn.functionArn
+      };
+    }
+
+    // Otherwise add rule to extract the response payload and use EventBridge
+    // as destination
+    if (!options) { // `options` added to bind() as optionnal to avoid breaking change
+      throw new Error('Options must be defined when using `responseOnly`.');
+    }
+
+    // Match invocation result of the function to which this destination is added and
```

Comment:
```suggestion
    // Match invocation result of the function to which destination this is added and
```

Reply:
Not a native speaker here, but are you sure? We are matching the invocation result of the "origin" function (the one with a destination configuration).

Reply:
How about 'Match invocation result of the source function and use it to trigger the destination function'?

Also, it's perfectly fine to drop this comment here.